.github/workflows/cta-sync-apply.yml

```
@@ -0,0 +1,94 @@
+# Apply a locally downloaded CTA sync tree to production D1.
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)